lib/chef/resource_inspector.rb

```diff
@@ -41,7 +41,11 @@ def self.extract_resource(resource, complete = false)
       data[:description] = resource.description
       # data[:deprecated] = resource.deprecated || false
       data[:default_action] = resource.default_action
-      data[:actions] = resource.allowed_actions
+      data[:actions] = {}
+      resource.allowed_actions.each do |action|
+        data[:actions][action] = resource.action_description(action)
+      end
```

Comment:
FWIW this could have been each_with_object, but it only saves a line at the cost of added overhead of knowing that method.